modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/BaseCartResourceImpl.java

```diff
@@ -190,10 +190,6 @@ public Cart patchCart(
 			existingCart.setAuthor(cart.getAuthor());
 		}
 
-		if (cart.getBillingAddress() != null) {
```

Comment:
Should not be removed as it has a ref that is not null

Reply:
it has to be removed, it's a schema